sql/commands/sql-alter-source.mdx

```diff
@@ -34,11 +34,46 @@ ALTER SOURCE src1
 ```
 
 <Note>
-* To alter columns in a source created with a schema registry, see [FORMAT and ENCODE options](/sql/commands/sql-alter-source#format-and-encode-options).
+* To alter columns in a source created with a schema registry, see [REFRESH SCHEMA](/sql/commands/sql-alter-source#refresh-schema).
 * You cannot add a primary key column to a source or table in RisingWave. To modify the primary key of a source or table, you need to recreate the table.
 * You cannot remove a column from a source in RisingWave. If you intend to remove a column from a source, you'll need to drop the source and create the source again.
 </Note>
 
+
+### `REFRESH SCHEMA`
```

Comment:
Moved up, since I think putting `ADD COLUMN` and `REFRESH SCHEMA` together is better.